### ISSUE TITLE
PDF: <fo:flow> empty because only <topigroup> as first child of opentopic:map

### ISSUE THREAD

**Issue by ColinMaudry** [dita-ot/dita-ot#1848]
Put simply, my map looks like this:

``` xml
<map xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" title="Package" xsi:noNamespaceSchemaLocation="urn:oasis:names:tc:dita:xsd:map.xsd">
    <topicgroup>
        <topicref id="package_information" href="package_information.dita"/>
        <topicref id="package_dimensions" href="package_dimensions.dita"/>
        <topicref id="package_outline" href="package_outline.dita"/>
    </topicgroup>
</map>
```

And when I process it with `-format pdf` in DITA OT 2.0, FOP complains that `fo:flow` is missing child elements and fails.

I know the `<topicgroup>` is a bit useless here, but we have still legacy non-DITA OT code that expects it.

I'm pretty sure the issue resides [here](https://github.com/dita-ot/dita-ot/blob/develop/src/main/plugins/org.dita.pdf2/xsl/fo/root-processing.xsl#L227), as doubling the slash solves the issue for me. I let you decide whether this is the right way to fix it.

**Comment by ColinMaudry** [comment]
I couldn't reproduce this issue with DITA OT 1.8 (the one bundled in Oxygen 16.1).

**Comment by dagoss** [comment]
I can confirm that this issue also occurs with OT2.0.1 and 2.1.0.

Using <topicgroup> in a map simply drops anything within it from the map. Adding the suggested change to root-process.xsl appears to fix the issue.

**Comment by robander** [comment]
Tagging as PDF bug - need to confirm if still exists in 2.2 / 2.3.

**Comment by raducoravu** [comment]
This still exists in DITA OT 2.x.
A DITA Map with content like:

        <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
        <map title="Growing Flowers">
          <topicgroup>
            <topicref href="topics/introduction.dita"/>
          </topicgroup>
        </map>

will fail to be published to PDF with an error like:

      org.apache.fop.apps.FOPException: org.apache.fop.fo.ValidationException: "fo:flow" is missing child elements. Required content model: marker* (%block;)+ (See position 1:-1)

**Comment by raducoravu** [comment]
Also one huge problem with DITA OT 2.3, all topicrefs located in <topicgroup>s are ignored in the PDF output. This properly worked with DITA OT 1.8.

**Comment by raducoravu** [comment]
Also one huge problem with DITA OT 2.3, all topicrefs located in <topicgroup>s are ignored in the PDF output. This properly worked with DITA OT 1.8.

**Comment by jelovirt** [comment]
Fixed by #2683
